tests/rbd/rbd_trash.py

```diff
@@ -30,9 +30,11 @@ def run(**kw):
     2. generate IO in images
     3. Enable "rbd_move_to_trash_on_remove" in client config
     4. Delete images and verify the deleted images are moved to trash automatically.
+    5. Disable "rbd_move_to_trash_on_remove" in client config and verify images are not moved to trash automatically
```

Comment:
Can we please update module doc with 
1) NEw Testcae id and summary
2) clarify in flow that 4th and 5th step are for different TCs ?